src/System.Data.OleDb/src/System/Data/Common/SR.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    internal static partial class SR
+    {
+        internal static string GetString(string value)
```

Comment:
Ah I see you worked around it with these glue functions. Fine, I don't feel strongly, although ideally you wouldn't need this file